src/session/content/chat_history.rs

```diff
@@ -20,40 +21,27 @@ mod imp {
     use once_cell::unsync::OnceCell;
     use std::cell::{Cell, RefCell};
 
-    #[derive(Debug, CompositeTemplate)]
+    #[derive(Debug, Default, CompositeTemplate)]
     #[template(resource = "/com/github/melix99/telegrand/ui/content-chat-history.ui")]
     pub(crate) struct ChatHistory {
         pub(super) compact: Cell<bool>,
         pub(super) chat: RefCell<Option<Chat>>,
+        pub(super) chat_handler: RefCell<Option<glib::SignalHandlerId>>,
         pub(super) message_menu: OnceCell<gtk::PopoverMenu>,
         pub(super) is_auto_scrolling: Cell<bool>,
         pub(super) sticky: Cell<bool>,
         #[template_child]
         pub(super) window_title: TemplateChild<adw::WindowTitle>,
         #[template_child]
+        pub(super) gradient_background: TemplateChild<GradientBackground>,
+        #[template_child]
         pub(super) scrolled_window: TemplateChild<gtk::ScrolledWindow>,
         #[template_child]
         pub(super) list_view: TemplateChild<gtk::ListView>,
         #[template_child]
         pub(super) chat_action_bar: TemplateChild<ChatActionBar>,
     }
 
-    impl Default for ChatHistory {
-        fn default() -> Self {
-            Self {
-                compact: Default::default(),
-                chat: Default::default(),
-                message_menu: Default::default(),
-                is_auto_scrolling: Default::default(),
-                sticky: Cell::new(false),
-                window_title: Default::default(),
-                scrolled_window: Default::default(),
-                list_view: Default::default(),
-                chat_action_bar: Default::default(),
-            }
-        }
-    }
-
```

Comment:
I cherry-picked this change in https://github.com/melix99/telegrand/commit/d08056d70cc0f59d953fc01bcb9acbf659087811.